### ISSUE TITLE
Improve Scala fetching

### ISSUE THREAD

**Issue by jackkoenig** [sifive/wit#46]
`fetch-scala` gives an incorrect error message asking for `ivydependency.json` instead of `ivydependencies.json`

Also, if you forget to `wit update` it may not find the file in a newly added package. This is a general problem for lots of stuff, in general, `wit` should probably warn if the lockfile and manifest are out of sync.

**Comment by aaronjanse** [comment]
Is this still an issue?

**Comment by jackkoenig** [comment]
Have we merged the warning if lockfile and manifest are out of date?

**Comment by aaronjanse** [comment]
Ah, I don't think so. I'd make a PR to do so, but I haven't used fetch-scala so I'm not sure how to tell if the json files are out of date

**Comment by jackkoenig** [comment]
Yeah I'm going to leave this open for the moment, I think there are 2 separate issues:

1. `wit status` should warn if `wit-lock` is out of date with `wit-manifest`
2. `wit status` should warn if `ivydependencies.json` are out of date with what has been fetched

**Comment by aaronjanse** [comment]
1. is already implemented, saying "{package name} will be added"
2. good idea. That could also allow us to always run `fetch-scala` for `wit update` without having to worry about speed
